### ISSUE TITLE
fix some languages' unreadable text

### ISSUE THREAD

**Issue by miaowwwwww** [GTNewHorizons/GT-New-Horizons-Modpack#15839]
### Your GTNH Discord Username

MIAOW246

### Your Pack Version

2.6.0 build 412

### Your Proposal

fix some languages' unreadable text
I think the render would just take place all together, i am gonna post some places which is important, so if it is fixed, we can check these places.
here i'm gonna post some screenshot that smooth font & Unicode font fixer does. which with angelica installed, not available rn.
without them:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/49871378/7845db02-2fbb-4e7e-9d5d-8c475758caf0)
with:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/49871378/d4b524fa-bd23-4ffc-8ff7-ab2092c1ec81)
without:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/49871378/9b45bf65-4d45-4d34-ac66-1e2aec81a9e7)
with:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/49871378/7f098fef-bb37-49bd-9a68-6ec01a4bcf1a)
(notice the "craft", one is unreadable, one is readable.)
without:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/49871378/eb1b39ab-18b9-407e-b28c-72c0c3f4ce9d)
with:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/49871378/9d5276a5-3cbf-4562-af44-e5b12c70ba53)

basically these are the most important places i can recall.

### Your Goal

make gtnh readable for all languages

### Your Vision

better gtnh

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.
